### ISSUE TITLE
[Feature Request] Provide shortcut customization for selecting chat by keyboard

### ISSUE THREAD

**Issue by tmeusburger** [telegramdesktop/tdesktop#1947]
The current UI provides the ability to change chats only via keyboard, but it took me a while of playing around before I was able to discern how.  This is provided more for others who may read this rather than maintainers, as I'm sure you're aware of the behavior.

Currently you can change the active chat by pressing escape, which closes your active chat and drops you in the search bar on the top left.  This allows you to then search by name/chat title and upon pressing enter will select the topmost chat (arrow keys can be used to select something other than the topmost).

It would be great if one of these two things were exposed via the UI/Settings:
1. Select a keybinding for the "select different chat".
2. Prevent the active chat from being closed until a new chat is selected and keeping escape for just closing the active chat.

Basically to provide a "quick swap chat" keyboard/action similar to other messaging apps.

**Comment by Aokromes** [comment]
#1257
